docs/develop/update/postgres.rst

```diff
@@ -48,14 +50,118 @@ Add your configuration
 
 The configuration file will be in the ``/etc/postgresql/11/main/conf.d/`` directory on the server (for PostgreSQL version 11).
 
+.. _pg-setup-backups:
+
+Set up backups
+--------------
+
+We use pgbackrest to create and manage backups stored in the cloud.
+Salt will install and configure pgbackrest if ``postgres:backup`` is defined in pillar.
+
+#. Create pgbackrest pillar config.
+
+   .. code-block:: yaml
+
+      postgres:
+        backup:
+          # The configuration file for pgbackrest, this is loaded from ``salt/postgres/files/pgbackrest/``.
+          config_name: kingfisher-process1
+          # Unique ID to store backups under. pgbackrest also refers to this as the "stanza".
+          identifier: kingfisher
+          # Concurrent processes for run pgbackrest with (backup speed vs CPU usage).
+          # Optional.
+          process_max: 4
+          # Backup bucket region.
+          region: eu-west-1
```

Comment:
I think we tend to use us-east.

Reply:
Happy to use one of the us-east regions instead of eu-west.
These three regions cost the same ($0.023 per GB).

It is a bit more expensive ($0.0245) but eu-central-1 (Frankfurt) is the closest to the Hetzner servers (hosted in Falkenstein and Helsinki). So for quicker backups and restorations this region would be better.

Reply:
I guess we should decide whether we will be using Backblaze instead?

Reply:
Happy to use an AWS EU region.

Reply:
Updated docs to `s3_region: eu-central-1` matching kingfisher settings.